lib/synced/strategies/full.rb

```diff
@@ -93,6 +90,10 @@ def perform
         end
       end
 
+      def reset_synced
+        # this is not supported by this strategy
```

Comment:
or we could just destroy all the records in the scope, that would do what the name of the method suggests :P